index.html

```diff
@@ -54,6 +54,197 @@ <h1 class="head">I’m Mariane<br>Glad to see<br> you!</h1>
         </ul>
 
     </section>
+    <div class="grid-container">
```

Comment:
- [ ] The "My Recent Work" title is missing. Please add it.

Reply:
I am not seeing a "My Recent Work" title in template 1

Reply:
- [ ] [OPTIONAL] since the first reviewer missed this point I marked it as optional but I really recommend you to do the change. To make your webpage more semantically correct, it's better to use the HTML semantic tag `<section>` instead of div for the works section.